docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,476 @@
+    - Support for changing rendering behavior has been added.
+    - A mechanism to specify a callback on every frame has been added.
+    - A CULLED property has been added to Actor.
+  - Application
+    - IdleEnterer callback has been added.
+  - Input
+    - An API has been added to identify right and left mouse button click.
+    - Support for enabling the text prediction has been added.
+    - Support for keyboard repeat information has been added.
+  - Text
+    - Support for software styling has been added.
+  - Timer
+    - API set to pause and resume the timer has been added.
+  - Control and Visual
+    - WebView control has been added.
+    - Support for FlexLayout, AbsoluteLayout and GridLayout has been added.
+  - 3D Model
+    - Support for GL Transmission Format (glTF) has been added.
+  - Performance
+    - Changed not to render transparent texts.
+  - NUI (C# Interface)
+    - Added support for WebView.
+- Simple DirectMedia Layer (SDL)
+  - Upgrade to 2.0.8.
+  - Added SDL sub-module as SDL-ttf.
+  - Added new features.
+    - Auxiliary Hint.
+    - Focus Skip.
+
+### Fixes
+
+- DALi (3D UI Toolkit)
+  - Actor and Renderer
+    - RenderTask bug to waiting forever has been fixed.
+    - Depth buffer clear bug has been fixed.
+    - BlendFunc has been fixed in case of a non-premultiplied format.
+  - Window
+    - Flickering issue has been fixed when a window is resized.
+  - Control
+    - Some bugs about layout have been fixed.
+  - NUI (C# Interface)
+    - Minor bugs (For example wrong position, not shown) of Layout have been fixed.
+    - RaiseToTop in View class bug has been fixed.
+    - XamlResource memory leak error has been fixed.
+    - VisualFittingMode bug in VisualMaps has been fixed.
+- Evas Rendering Engine.
+  - Bug of rotation is fixed in Evas GL Engine.
+  - Added thread safety patch to Evas TBM Engine.
+  - Bug of image downscaling is fixed in Evas GL Engine.
+  - Added the patch is TBM Queue is used per TBM surface in Evas TBM Engine.
+-  Simple DirectMedia Layer (SDL)
+  - Bug of pause/resume in multi-windows is fixed.
+
+
+## UI framework
+
+### New and changed features
+
+- EFL
+  - Version 1.20 is upgraded to 1.21.
+- Harfbuzz
+  - Version 0.4.6 is upgraded to 1.8.1.
+- Freetype2
+  - Version 2.9 is upgraded to 2.9.1.
+- Text Input
+  - Natural Language Processing (NLP) API has been added.
+- Voice interaction
+  - Added new API set to support multiple assistant.
+  - Added API set, which has platform privileges to support voice panel app. 
+  - Added API set to support sending and receiving specific engine result.
+
+### Fixes
+
+- Fixed bugs.
+- Clip Board History Manager (CBHM) is now working properly.
+
+
+## Multimedia framework
+
+### New and changed features
+
+- Common
+  - Tizen Allocator and Bufferpool has been added.
+- Media Player
+  - Playback time related API set in nanoseconds has been added.
+  - Setting zoom level with fov about spherical video API has been added.

```

Comment:
Is it fov or FOV?

Reply:
fov => FOV == filed of view
will use FOV instead of fov.

Reply:
Thank you. Please use FOV and please provide the full form also.

Rephrase as:
field of view (FOV)

Reply:
will rephrase as:
Setting zoom level with Field of View (FOV) for spherical video API has been added.

Reply:
Yes.